src/win32/map.c

```diff
@@ -99,7 +99,7 @@ int p_mmap(git_map *out, size_t len, int prot, int flags, int fd, git_off_t offs
 		return -1;
 	}
 
-	assert(sizeof(git_off_t) == 8);
+	assert(sizeof(off64_t) == 8);
```

Comment:
Does this make sense? I mean it's very unlikely that `off64_t` is ever going to be something else than 8 bytes

Reply:
😬 Yes, that's very true.  Removed entirely.